extensions/EXT_disjoint_timer_query/extension.xml

```diff
@@ -214,8 +214,8 @@ interface EXT_disjoint_timer_query {
   <samplecode xml:space="preserve">
     <pre>
         // Example (1) -- uses beginQueryEXT/endQueryEXT.
-        var ext = gl.getExtension('EXT_disjoint_timer_query');
-        var query = ext.createQueryEXT();
+        let ext = gl.getExtension('EXT_disjoint_timer_query');
```

Comment:
I think most of these `let`s could be `const`s instead (though this is a less important improvement than `var` &rarr; `let`).

Reply:
Yes, probably - I looked at doing that but didn't want to make mistakes and also didn't really have the time to copy out the example code and double-check my work.

Reply:
At this point the query variables are all mutable, so can't be const. The extension objects could be. Let me know if you'd prefer I make that change throughout.

Reply:
I don't think it's important.